paddle/fluid/framework/details/eager_deletion_op_handle.cc

```diff
@@ -119,6 +120,7 @@ void EagerDeletionOpHandle::RunImpl() {
     auto *var_info = var_infos_[i];
     if (var_info->IsSkippedAllMemoryOptimization() ||
         !var_info->DecreaseRefCnt()) {
+      VLOG(1) << "skip memory optimization with var: " << var_info->Name();
```

Comment:
Suggest to use VLOG(4) as what we did for variable information in the past.

Reply:
Thanks, fixed